components/next_steps_view/next_steps_view.tsx

```diff
@@ -154,11 +153,31 @@ export default class NextStepsView extends React.PureComponent<Props, State> {
         );
     }
 
-    renderFinalScreen = () => {
-        // TODO
+    renderTransitionScreen = () => {
         return (
-            <div>
-                {'Placeholder for Final Screen'}
+            <div
+                className={classNames('NextStepsView__viewWrapper NextStepsView__transitionView', {
+                    transitioning: this.state.showTransitionScreen,
+                    completed: this.state.showTransitionScreen && this.state.showFinalScreen,
+                })}
+                onTransitionEnd={this.setTimerToFinalScreen}
```

Comment:
is this reliable? I remember reading that `onTransitionEnd` can sometimes ignore the transitions of the child elements.

Reply:
Yeah it should be fine, I only care about transition on that `div` anyways for that callback.